mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/RabbitMQEventBusPublishingTest.java

```diff
@@ -108,6 +115,32 @@ void dispatchShouldPublishSerializedEventToRabbitMQWhenNotBlocking() {
         assertThat(dequeueEvent()).isEqualTo(EVENT);
     }
 
+    @Test
+    void startShouldCreateEventExchange() {
+        assertThat(rabbitMQManagement.listExchanges())
+            .filteredOn(exchange -> exchange.getName().equals(MAILBOX_EVENT_EXCHANGE_NAME))
+            .hasSize(1);
+    }
+
+    @Test
+    void startShouldNotThrowWhenStartingMultipleEventBus() {
```

Comment:
move to SeveralRabbitMQEventBusTest

Reply:
You can maybe try with a simple eventBus:

 - eventBus.start()
 - eventBus.stop()
 - eventBus.start()
 - eventBus.stop()
 - eventBus.start()
 - eventBus.stop()